claripy/backends/backend_z3.py

```diff
@@ -309,15 +309,15 @@ def FPS(self, ast):
     def FPV(self, ast):
         val = str(ast.args[0])
         sort = self._convert(ast.args[1])
-        if val in ("+oo", "+inf", "+Inf", 'inf'):
+        if val == "+oo" or val == "+inf" or val == "+Inf" or val == 'inf':
```

Comment:
Why these changes, out of curiosity?

Reply:
Ops those are wrong changes which slipped in during the merge with the branch `wip/z3_updates`... Fixed now.